packages/ceramic-doctype-tile/src/__tests__/three-id.test.ts

```diff
@@ -81,7 +81,7 @@ const RECORDS = {
         ],
         "prev": "bafybeig6xv5nwphfmvcnektpnojts33jqcuam7bmye2pb54adnrtccjlsu",
         "header": {
-          "nonce": 0
+          "nonce": undefined
```

Comment:
Seems like the nonce is still being set here?

Reply:
then I'll remove `nonce` field whatsoever